backend/app/Http/Resources/AvgSession.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use App\Contracts\ApiResponse;
+
+class AvgSession extends JsonResource implements ApiResponse
+{
+    public function toArray($request): array
+    {
+        return [
+            'avg_session' => $this->convertSecondsToHoursMinutesSeconds($this->resource)
+        ];
+    }
+
+    private function convertSecondsToHoursMinutesSeconds(int $seconds): array
```

Comment:
Better return timestamp in UTC, and UI will make a decision how to convert it to appropriate format